v1/python-sdk/tutorials/automl-with-azureml/forecasting-demand-forecasting-many-models/auto-ml-forecasting-demand-forecasting-many-models.ipynb

```diff
@@ -0,0 +1,1626 @@
+    "| **train_run_id**                   | \\[Optional\\] The run id of the **training pipeline**. By default it is the latest successful training pipeline run in the experiment. |\n",
+    "| **train_experiment_name**          | \\[Optional\\] The train experiment that contains the train pipeline. This one is only needed when the train pipeline is not in the same experiement as the inference pipeline. |\n",
+    "| **run_invocation_timeout**         | \\[Optional\\] Maximum amount of time in seconds that the `ParallelRunStep` class is allowed. This is optional but provides customers with greater control on exit criteria. |\n",
+    "| **output_datastore**               | \\[Optional\\] The `Datastore` or `OutputDatasetConfig` to be used for output. If specified any pipeline output will be written to that location. If unspecified the default datastore will be used. |\n",
+    "| **arguments**                      | \\[Optional\\] Arguments to be passed to inference script. Possible argument is '--forecast_quantiles' followed by quantile values. |"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from azureml.contrib.automl.pipeline.steps import AutoMLPipelineBuilder\n",
+    "from azureml.train.automl.runtime._many_models.many_models_parameters import (\n",
+    "    ManyModelsInferenceParameters,\n",
+    ")\n",
+    "\n",
+    "output_file_name = \"parallel_run_step.csv\"\n",
+    "\n",
+    "mm_parameters = ManyModelsInferenceParameters(\n",
+    "    partition_column_names=time_series_id_column_names,\n",
+    "    time_column_name=time_column_name,\n",
+    "    target_column_name=target_column_name,\n",
+    ")\n",
+    "\n",
+    "inference_steps = AutoMLPipelineBuilder.get_many_models_batch_inference_steps(\n",
+    "    experiment=experiment,\n",
+    "    inference_data=test_dataset,\n",
+    "    node_count=2,\n",
+    "    process_count_per_node=8,\n",
+    "    compute_target=compute_target,\n",
+    "    run_invocation_timeout=300,\n",
+    "    output_datastore=output_test_data_ds,\n",
+    "    train_run_id=training_run.id,\n",
+    "    train_experiment_name=training_run.experiment.name,\n",
+    "    inference_pipeline_parameters=mm_parameters,\n",
+    "    append_row_file_name=output_file_name,\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from azureml.pipeline.core import Pipeline\n",
+    "\n",
+    "inference_pipeline = Pipeline(ws, steps=inference_steps)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "scrolled": true,
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "inference_run = experiment.submit(inference_pipeline)\n",
```

Comment:
Let us add the cell and say "Run the pipeline"

Reply:
done